prime-router/build.gradle.kts

```diff
@@ -827,7 +827,7 @@ dependencies {
         exclude(group = "org.yaml", module = "snakeyaml")
     }
     implementation("org.yaml:snakeyaml:2.2")
-    implementation("io.github.linuxforhealth:hl7v2-fhir-converter:1.0.19")
+    implementation("com.github.focusconsulting:hl7v2-fhir-converter:v1.0.20-SNAPSHOT")
```

Comment:
This takes advantage of jitpack.io which will dynamically package a GH repo into a jar.  Jitpack.io was originally added to download fuzzy-csv which was then subsequently removed.   This would be a new instance of of using the tool.

Reply:
It looks like com.github.KennethWussmann:mock-fuel also relies on jitpack

Reply:
@snesm Yes this is publicly accessible and also temporary while we wait for the CDC machinery to create a repo in the org when it can get hosted.